### PR TITLE
ridesharing/ when user registers in auth api, notify ridesharing service too

### PR DESCRIPTION
**Pull request by NafisianCastle**
Issue Link : [https://github.com/sayeed1999/Ride-Sharing-Platform/issues/5](https://github.com/sayeed1999/Ride-Sharing-Platform/issues/5)

### CODE REVIEW

File: AuthService.API/Controllers/UserController.cs [hunk 1]
```diff
@@ -15,7 +15,8 @@
 
 namespace AuthService.API
 {
-    [Authorize(Policy = AuthorizationPolicies.AdminOnly)]
+    //[Authorize(Policy = AuthorizationPolicies.AdminOnly)]
```
Comment (sayeed1999) [inline AuthService.API/Controllers/UserController.cs:18]: take authorize to previous step

File: BlogService.API/appsettings.json [hunk 2]
```diff
@@ -7,11 +7,11 @@
   },
   "AppSettings": {
     "ConnectionStrings": {
-      "ConnStr": "Server=sayeed_1999; Database=blog_service_db; trusted_connection=SSPI; TrustServerCertificate=True; MultipleActiveResultSets=True"
```
Comment (sayeed1999) [inline BlogService.API/appsettings.json:10]: undo it

File: AuthService.API/Controllers/UserController.cs (hunk 1 above)
Comment (NafisianCastle) [reply AuthService.API/Controllers/UserController.cs]: done

File: BlogService.API/appsettings.json (hunk 2 above)
Comment (NafisianCastle) [reply BlogService.API/appsettings.json]: done

**Review by sayeed1999 — APPROVED** [review]

**Pull request merged**